graph-db/src/utils/indexing/indexRelatedUrl.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Given a RelatedUrl object, Gremlin connection, and
+ * associated collection, index related URL and build
+ * relationships for any GENERAL DOCUMENTATION that exists
+ * in common with other nodes in the graph database
+ * @param {JSON} relatedUrl a list of RelatedUrls
+ * @param {Connection} gremlin a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+exports.indexRelatedUrl = (relatedUrl, gremlin, dataset) => {
+  const {
+    Type: type,
+    SubType: subType,
+    URL: url,
+    Description: description,
+  } = relatedUrl
+  if (!!type
+        || !!subType
+        || type !== 'VIEW RELATED INFORMATION'
```

Comment:
Subtypes are not required, even for documentation - so you can be missing out on a lot of documentation files if you filter out by missing subtype.